app/controllers/api/v1/annotation_documents_controller.rb

```diff
@@ -173,7 +174,11 @@ def create
 
       # GET /api/v1/annotation_documents/1
       def show
-        render json: @annotation_document.relevant_attributes
+        if @annotation_document.update(requested_at: Time.zone.now)
```

Comment:
`record.update(requested_at: Time.zone.now)` is perfect but perhaps we should make usage of http's PATCH method for this action as it alters a record.